src/lightning/app/components/serve/theme.py

```diff
@@ -0,0 +1,101 @@
+from types import ModuleType
+
+from lightning.app.utilities.imports import _is_gradio_available
+
+if not _is_gradio_available():
+    theme = ModuleType("theme")
+else:
+    import gradio
+
+    theme = gradio.themes.Default(
```

Comment:
I propose to put this definition into a function

`theme = lightning_gradio_theme()`

and move below the ServeGradio class definition